senza/cli.py

```diff
@@ -116,6 +116,13 @@
 }
 
 
+def filter_output_columns(output_columns, filter_columns=list()):
```

Comment:
True, I don't need a default value at all as I always expect this function to be called with 2 arguments.